build.gradle

```diff
@@ -111,6 +111,38 @@ static Map<String, String> getEnvironmentVariables(String... envNames) {
     return envNames.collectEntries {envName -> [(envName): System.getenv(envName)] }
 }
 
+String getMavenRepositoryURL() {
+    def url = "${environment.OBLIVIUM_MAVEN_URL}/${environment.OBLIVIUM_MAVEN_REPOSITORY}"
```

Comment:
Hardcode the public repo URLs, as external developers will need to know them anyway.